### ISSUE TITLE
Update "killed by" images.

### ISSUE THREAD

**Issue by crescentrose** [thronebutt/thronebutt-bugs#6]
Change the "killed by" images for the following: 
- Big Dog's explosion (ID: 102)
- hostile Horror (ID: 60)
- reflected projectile off a shielder's shield (it shows Crystals shield atm) (ID: 58)
- Cursed ammo (does it have one yet?) (ID: 87)
- Crown of Hatred (does it have one yet?) (ID: 85)
- lightning in Oasis/underwater lightning (ID: 88) (does it have one yet?)

**Comment by Stef-Verzelen** [comment]
After further inspection not only the "Big Dog's explosion (ID: 102)" image is wrong, but also the "Big Dog (ID: 18)" image is wrong. So both need to be changed. "Big Dog (ID: 18)" currently uses the image that's meant to be for "Big Dog's explosion (ID: 102)" that shows him exploding. And for "Big Dog (ID: 18)" you'll need a new image that shows him in his idle animation.

TL;DR: Both "Big Dog's explosion (ID: 102)" and "Big Dog (ID: 18)" need a new image.

**Comment by Stef-Verzelen** [comment]
Image for getting killed by a reflected projectile of a Shielder's shield (ID: 58) is changed and correct now, but it's way too large (scaled up too much). Also maybe remove the Shielder inside and just have the shield animation itself like it shows in-game?

**Comment by crescentrose** [comment]
I pulled stuff from the game, can't really change the shielder animation myself (way too bad with pixel art for that)
